### PR TITLE
Accelo #3533

### PR DESCRIPTION
**Pull request by lcaresia**

### CODE REVIEW

File: components/accelo/accelo.app.mjs [hunk 1]
```diff
@@ -1,11 +1,214 @@
...
+    }) {
+      return axios($, {
+        url: `${this._apiUrl()}${path}`,
+        headers: {
+          Authorization: `Bearer ${this._accessToken()}`,
+        },
+        ...args,
+      });
+    },
+    async createWebhook({ ...args }) {
```
[the remark this answers is not in the document]
Comment (lcaresia) [reply components/accelo/accelo.app.mjs:112]: I read the discussion topic on Slack about this, but for now I prefer to put all in the app file (as I was doing) and just change when we have an "official" decision about this, what you think?

**Comment by DilanAthukorala** [comment]
@lcaresia Please check the following,

1. Create_Contact & Update_Contact Actions - Prop Name and Description should be corrected to "Surname"
image


2. Create Contact Action - firstname and surname fields are mandatory parameters

3. Create_Prospect Action - The prospect type ID is a mandatory parameter

4. New Request Created and New Task Assigned Source - Does not emit events
image

**Comment by DilanAthukorala** [comment]
Verification completed, ready for release!

**Pull request merged**